colour/models/rgb/transfer_functions/tests/tests_red_log.py

```diff
@@ -340,22 +340,22 @@ def test_log_encoding_Log3G10(self):
         """
 
         self.assertAlmostEqual(
-            log_encoding_Log3G10(0.0),
+            log_encoding_Log3G10(0.0, legacy_curve=True),
```

Comment:
To cover all the possibilities we should have tests for positive and negative values for both versions of the curve, and also nan/inf tests for both versions.